Project.toml

```diff
@@ -10,7 +10,7 @@ FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 
 [compat]
-ChainRulesCore = "0.9"
+ChainRulesCore = "0.9.44, 0.10"
```

Comment:
```suggestion
ChainRulesCore = "0.9.44, 0.10.1"
```
if we want to keep the compat with 0.9

Reply:
It doesn't actually use `NO_FIELDS` so i think we are fine.
(technically we are not fine, and this could breaking things since it changed the first return from `ZeroTangent()` to `NoTangent`. But I think we will get away with it.)